src/Storage/ServiceConfiguration.ts

```diff
@@ -1,3 +1,5 @@
 export interface ServiceConfiguration {
   readonly downloadIntervalInSeconds: number
+  readonly downloadRetryDelayInMinutes: number
```

Comment:
Please create a ControllerConfiguration for these, following the same pattern used by BlockchainReader for example